### PR TITLE
SED-2931 fix paths

### PR DESCRIPTION
**Pull request by lukasz-lepa**
https://exense.atlassian.net/browse/SED-2931

### CODE REVIEW

**Comment by dvladir** [comment]
@neogucky I would recommend to recheck it, after https://github.com/exense/step-frontend/pull/755 will be merged

**Review by neogucky — DISMISSED** [review]

File: projects/step-frontend/src/lib/modules/admin/admin.module.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ export class AdminModule {
 
     this._viewRegistry.registerRoute(
       {
-        path: 'myaccount',
+        path: 'my-account',
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/admin/admin.module.ts:116]: @lukasz-lepa 
There are still places in project, that references to `myaccount` link

File: projects/step-frontend/src/lib/os-plugins/modules/function-packages/function-packages.module.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ export class FunctionPackagesModule {
 
   private registerViews(): void {
     this._viewRegistry.registerRoute({
-      path: 'functionPackages',
+      path: 'function-packages',
```
Comment (dvladir) [inline projects/step-frontend/src/lib/os-plugins/modules/function-packages/function-packages.module.ts:77]: There are places in project, that still references to `functionPackages` link

**Review by dvladir — CHANGES_REQUESTED** [review]

File: projects/step-frontend/src/lib/modules/admin/admin.module.ts (hunk 1 above)
Comment (lukasz-lepa) [reply projects/step-frontend/src/lib/modules/admin/admin.module.ts:116]: Fixed.

File: projects/step-frontend/src/lib/os-plugins/modules/function-packages/function-packages.module.ts (hunk 2 above)
Comment (lukasz-lepa) [reply projects/step-frontend/src/lib/os-plugins/modules/function-packages/function-packages.module.ts:77]: Fixed.

**Review by dvladir — APPROVED** [review]

**Pull request merged**